internal/resources/managementclusterregistration/constants.go

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package managementclusterregistration
+
+const (
+	NameKey = "name"
+
+	specKey            = "spec"
+	registerClusterKey = "register_management_cluster"
+
+	registerClusterKubeConfigPathForTKGmKey = "kubeconfig_file"
```

Comment:
Can we update the value here, as the customer will use the value and not the variable name using the provider?

kubeconfig_file => `tkgm_kubeconfig_file`

Same comment for `registerClusterKubeConfigRawForTKGmKey  = "kubeconfig_raw"`

Reply:
Done.